config.py

```diff
@@ -30,6 +33,9 @@ class Config(object):
     PASSWORD_MIN_LENGTH = 8
     PASSWORD_MAX_LENGTH = 160
 
+    OAUTH_SERVICE_URL = os.getenv('OAUTH_SERVICE_URL', 'http://localhost:8040')
```

Comment:
Don't think these should have defaults here only in the dev config.

Reply:
@gemmai95 I agree. And would be consistent with the new r-ops changes.